VocaDbModel/DataContracts/ReleaseEvents/ReleaseEventForApiContract.cs

```diff
@@ -103,6 +103,8 @@ public class ReleaseEventForApiContract : IReleaseEvent, IEntryBase {
 		[DataMember]
 		public DateTime? EndDate { get; set; }
 
+		public bool HasVenueOrVenueName => Venue != null || !string.IsNullOrEmpty(VenueName);

```

Comment:
Is it possible for an API contract to have read-only properties?

Reply:
There should be no problem with that.